views/summary.html

```diff
@@ -20,50 +20,9 @@
 <body>
   <div class="container">
 
-    <!-- if top or saved, show those -->
```

Comment:
Did you mean to remove all this?

Reply:
haha no i didn't. thank you for catching this